jest.int.config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "setupFiles": [
+    "<rootDir>/src/util/globals.js"
+  ],
+  "moduleFileExtensions": [
+    "js",
+    "jsx"
+  ],
+  "testRegex": "tests_integration\/.*\\.int\\.jsx?$"
```

Comment:
if we're ever going to have other types of tests in a test directory, i'd opt for `tests/integration/` here instead.